pkg/provider/github/app/token.go

```diff
@@ -102,17 +102,16 @@ func (ip *Install) GetAndUpdateInstallationID(ctx context.Context) (string, stri
 	return enterpriseHost, token, installationID, nil
 }
 
-func (ip *Install) listRepos(ctx context.Context) (bool, error) {
-	if ip.repoList == nil {
-		var err error
-		ip.repoList, err = github.ListRepos(ctx, ip.ghClient)
-		if err != nil {
-			return false, err
-		}
+// matchRepos check for matching repo and update the instance repoList.
```

Comment:
repos is a bit overlooked term, any chance you can adjust the comment to specify matching github repositories to its installation ids.

Reply:
Sure. Updated.